src/ctia/http/routes/crud.clj

```diff
@@ -181,12 +185,14 @@
                                services
                                entity-crud-config)
                              routes.common/prep-sort_by-param-schema)
-        search-filters (st/dissoc search-q-params
-                                  :sort_by
-                                  :sort_order
-                                  :fields
-                                  :limit
-                                  :offset)
+        search-filters (apply st/dissoc search-q-params
+                              :sort_by
+                              :sort_order
+                              :fields
+                              :limit
+                              :offset
+                              ;; TODO support extensions in non-"search" aggregation routes
+                              (mapcat keys [sort-extension-definitions]))
```

Comment:
we do not need to sort in aggregations (we just return the aggregation of matched documents)

Reply:
Ah!